pkgs/ffigen/lib/src/code_generator/objc_block.dart

```diff
@@ -275,6 +275,7 @@ $blockTypedef $fnName($blockTypedef block) {
         objCReturnsRetained: true,
         isLeaf: true,
         isInternal: true,
+        originalNameIsRenamable: true,
```

Comment:
I wonder if this is the right approach.

Conceptually any original names (from the source language) should not be renameable, because looking up things would fail.

Here we actually bind to some native code that we generate, so I would maybe not call that the original name.

Is there a way we can separate the concepts of (1) a name in the native source code "originalName" and (2) a name in the native source code that we generate "intermediateName". (And then "name" for what the user-uses.)

Reply:
We'd need a flag in `Func` to tell it whether to use the `originalName` or `intermediateName` when looking up the symbol. When considering that option I realized we can just add a flag that tells the `Func` to use the `name` instead of the `originalName` for lookup, and then we wouldn't need any extra renaming steps in the library.